### PR TITLE
Fixes #60 Enabling in parameter "language" of the command "applySettings"

### LINKED ISSUE #60: Apply settings language not used
**Describe the bug**
`ApplySettings` does not take `language` parameter

**Environment information:**
- Any

**Expected behavior**
Add `language` parameter in `ApplySettings` function

### PR DESCRIPTION
**Pull request by stdevHan**
Fixes #60 

Changes:
- Adding the `language` parameter to the `applySettings` command.

Does this change need to mentioned in CHANGELOG.md?
yes

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,7 @@
 DS_Store
 
 vendor/github.com/google/protobuf/descriptor.pb.go
+
+#IDE
+.idea
+.vscode
```
Comment (therealssj) [inline .gitignore]: new line

File: cmd/cli/README.md [hunk 2]
```diff
@@ -101,6 +120,26 @@ Success with code:  Settings applied
...
+Configure device with settings such as: change language
+
+
+```bash
+$ skycoin-hw-cli applySettings --language en
+```
+
+```
+OPTIONS:
+        --language string            Configure a device language
```
Comment (therealssj) [inline cmd/cli/README.md]: missing `label`, `usePassphrase` and `deviceType` options

**Comment by olemis** [comment]
@Senyoret1 can we postpone this ? Looks like v2 to me but ... do you need it for v1 ?

**Comment by Senyoret1** [comment]
I do not need it for v1. This would be necessary for V1 only if there is a problem or bug when not sending that parameter to the firmware.

**Comment by therealssj** [comment]
this is a very simple change and should not take more than an hour or two to fix. lets just get this done with since its already done 99% finished. 

postponing this means we come back to it later and then rebasing which is just a waste of time.

if u want you can close this PR here and i can take it over.

**Comment by olemis** [comment]
@therealssj The patch is there . I just wanted to know if it was worth reviewing . I'll take a look .

**Pull request merged**